src/Api/AbstractApi.php

```diff
@@ -10,9 +10,12 @@
 abstract class AbstractApi
 {
     protected Rest $restClient;
+    protected string $baseUrl;
 
     public function __construct(protected readonly AbstractClient $client)
     {
         $this->restClient = $this->client->getClient();
+        $url              = $this->client->getBaseUrl();
+        $this->baseUrl    = str_ends_with($url, '/') ? $url : $url.'/';
```

Comment:
This should be `AbstractClient`'s responsibility. Just use `$this->client->getBaseUrl();` directly when you need to.